src/ContainerRegistry/ContainerRegistry/ChangeLog.md

```diff
@@ -18,6 +18,16 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added cmdlets:
```

Comment:
Supported repository, manifest, and tag operations
- cmdlet lists

Reply:
Supported login to container registry through `Connect-AzContainerRegistry`

Reply:
it was added from previous release

Reply:
> Supported login to container registry through `Connect-AzContainerRegistry`

it was added in previous release